packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -64,8 +65,13 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
 
   protected selection: Selection;
   protected suggestionStore: SuggestionsController<T>;
+  /**
+   * @deprecated this is no longer necessary as typescript now supports generic elements
+   */
+  protected SuggestionOfProperType = Suggestions as new (props: ISuggestionsProps<T>) => Suggestions<T>;
```

Comment:
Is there a reason we can't avoid the cast and do 
`protected SuggestionOfProperType: (new (props: ISuggestionsProps<T>) => Suggestions<T>) = Suggestions;`?

Reply:
Cool! That works fine it seems.  That said I think we should keep it this way because that's the way it was and I think it should be as close to possible to that since we'll be removing it eventually.